activities/views/posts.py

```diff
@@ -200,12 +211,21 @@ def clean_text(self):
                 )
             return text
 
+        def __init__(self, *args, **kwargs):
+            super().__init__(*args, **kwargs)
+            initial = kwargs.get("initial")
+            if initial.get("reply_to"):
+                # Maintain local-ness of threads
+                self.fields["local_only"].disabled = True
+
     def get_initial(self):
         initial = super().get_initial()
+        initial["local_only"] = self.request.identity.config_identity.local_only_default
```

Comment:
I don't like the the name of `config_identity` specifically because of the redundant `identity`, but naming it `config` is too close to typo realm of the reverse `configs` for my liking.